scale/job/models.py

```diff
@@ -3503,6 +2589,7 @@ def _validate_job_type_fields(self, **kwargs):
         :raises :class:`job.exceptions.InvalidJobField`: If a given job type field has an invalid value
         """
 
+        # TODO: remove this timeout check? - Mike
         if 'timeout' in kwargs:
```

Comment:
The _validate_job_type_fields method can be removed whole cloth not just the timeout as it is only used by v5 methods. V6 does not use kwargs but only specified args.